### ISSUE TITLE
[IDEA] Display Name of the Repo and Version Number

### ISSUE THREAD

**Issue by andymcfee** [artemave/StarLogs#4]
Start by showing the name of the Repo in the same way STAR WARS is the first thing to fly back.  Then, the first scroll item should be the Version Number.

An example using BootStrap
STAR WARS  =  Bootstrap
EPISODE IV  =  Version 2.3.1
A NEW HOPE =  By Twitter

Or something like that.

**Comment by endreman0** [comment]
Maybe the first line is the repository name, followed by the description? Example for [your "cccv" repo](https://github.com/artemave/cccv):

cccv
Finds copy/pastes in a git diff 

Update README.md
better regexp example in readme
Merge pull request #1 from gasche/patch-1
reading files in goroutines does not work

etc.

**Comment by artemave** [comment]
Still not sure about it. 

What if tomorrow you'd want to check what new commits appeared on cccv? You wouldn't want to see the description again, would you?

**Comment by endreman0** [comment]
True, but seeing the repository name and description seems more like the official *Star Wars* style.

**Comment by artemave** [comment]
That is a good point.

**Comment by elifoster** [comment]
I support this minimally. I think the repository name would fit and look nice, however I don't think displaying versions or descriptions would fit. Maybe the author name wouldn't be so bad, though.
